srcpkgs/liburing/template

```diff
@@ -11,7 +11,11 @@ license="LGPL-2.1-only"
 homepage="https://git.kernel.dk/cgit/liburing"
 changelog="https://git.kernel.dk/cgit/liburing/plain/CHANGELOG"
 distfiles="https://git.kernel.dk/cgit/liburing/snapshot/liburing-${version}.tar.gz"
-checksum=a65a6adbe80425c1c4d0740532ba42c3d4fd9dadd17a0e0bfd31c29e1c14dba8
+checksum=b5fbb1f6d65d88a8b37cbf3f5d4a7fa38d1d9e928a83df79c6469ed816bbcacc
+
+do_check() {
+	make -C test
```

Comment:
could this be done with `make_check_args` or `make_check_target`?

Reply:
yeah but it's two lines though
```
make_check_args="-C"
make_check_target="test"
```